modules/examiner/README.md

```diff
@@ -0,0 +1,91 @@
+# Examiner
+
+## Purpose
+
+The Examiner module displays the list of examiners in the study and allows 
+for additions and removal of certifications for each examiner listed.
+
+## Intended Users
+
+The primary type of users is:
+1. Site coordinators adding examiners and adding certifications to examiners
+
+
+## Scope
+
+This module displays information about examiners as well as certifications and 
+sites to which the examiners are affiliated. It also allows adding examiners 
+to sites, to give or remove certifications for instruments and set them as 
+radiologists.
+
+NOT in scope:
+
+Management of the certification instruments and of the sites is external to 
+this module.
+
+The Examiner architecture is completely separate from the User 
+architecture intentionally; examiners may or may not be users.
+
+
+## Permissions
+
+The Examiner module uses the following permissions. Any one of them 
+is sufficient to have access to the module.
+
+examiner_view
+ - This permission gives the user viewing and editing access to all 
+ examiners and certifications within their own site
+
+examiner_multisite
+ - This permission gives the user viewing and editing access to all 
+ examiners and certifications at all sites.
+
+## Configurations
+
+The examiner module has the following configurations that affect its usage
+
+EnableCertification (Config.xml)
+ - Binary entry for sub-tag `<EnableCertification>` under the tag 
+ `<Certification>` in the config.xml file. Options `1` or `0` respectively 
+ enables or disables the use of certification for projects defined in the 
```

Comment:
"respectively" needs to come after the referents.

(I don't know why, the closest I can find to an explanation is the assertions in the answers on https://english.stackexchange.com/questions/138695/when-to-use-respectively ..)

Reply:
@driusan not sure if I made it better or worse